src/toast/ops/flag_sso.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) 2021 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+from time import time
+
+from ..mpi import MPI
+
+import traitlets
+
+import numpy as np
+
+from astropy import units as u
+
+import ephem
+
+import healpy as hp
+
+from ..timing import function_timer
+
+from .. import qarray as qa
+
+from ..data import Data
+
+from ..traits import trait_docs, Int, Unicode, Bool, Quantity, Float, Instance, List
+
+from .operator import Operator
+
+from .pipeline import Pipeline
+
+from ..utils import Environment, Logger, Timer
+
+from ..observation import default_names as obs_names
+
+
+def to_JD(t):
+    # Unix time stamp to Julian date
+    # (days since -4712-01-01 12:00:00 UTC)
+    return t / 86400.0 + 2440587.5
+
+
+def to_MJD(t):
+    # Convert Unix time stamp to modified Julian date
+    # (days since 1858-11-17 00:00:00 UTC)
+    return to_JD(t) - 2400000.5
```

Comment:
These are now defined in multiple operator source files.  Maybe they should go to some place that can be imported by all the operators using them?